### PR TITLE
changing the list of filters for RTE

### PR DESCRIPTION
**Pull request by AdamDM**

### CODE REVIEW

File: src/main/java/org/atlasapi/equiv/EquivModule.java [hunk 1]
```diff
@@ -625,6 +643,30 @@ private SourceSpecificEquivalenceUpdater roviUpdater(Publisher roviSource, Immut
             .build();
     }
 
+    private EquivalenceUpdater<Container> rteVodContainerUpdater(Set<Publisher> acceptablePublishers) {
+        return ContentEquivalenceUpdater.<Container> builder()
+                .withExcludedUris(excludedUrisFromProperties())
+                .withGenerator(TitleSearchGenerator.create(searchResolver, Container.class, acceptablePublishers, DEFAULT_EXACT_TITLE_MATCH_SCORE)
```
Comment (jakekeeys) [inline src/main/java/org/atlasapi/equiv/EquivModule.java]: Line length

**Comment by jakekeeys** [comment]
OLGTM

File: src/main/java/org/atlasapi/equiv/EquivModule.java [hunk 2]
```diff
@@ -625,6 +643,30 @@ private SourceSpecificEquivalenceUpdater roviUpdater(Publisher roviSource, Immut
...
+                                Score.negativeOne(),
+                                new SubscriptionCatchupBrandDetector(contentResolver)
+                        )
+                ))
+                .withCombiner(new RequiredScoreFilteringCombiner<Container>(
+                        new NullScoreAwareAveragingCombiner<Container>(),
+                        TitleMatchingContainerScorer.NAME,
+                        ScoreThreshold.greaterThanOrEqual(DEFAULT_EXACT_TITLE_MATCH_SCORE))
+                )
+                .withFilter(this.filtersForRTE())
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/equiv/EquivModule.java]: `this.` is unnecessary

File: src/main/java/org/atlasapi/equiv/EquivModule.java (hunk 1 above)
Comment (Dalamar42) [inline src/main/java/org/atlasapi/equiv/EquivModule.java]: That ^

**Pull request merged**